Makefile

```diff
@@ -45,17 +46,29 @@ endif
 build:
 ifeq ($(OS),Windows_NT)
 	make genproto
-	go build ${LDFLAGS} -o $(CURR_DIR_WIN)/$(BINARY).exe
+	go build ${LDFLAGS} -o $(BIN_DIR_WIN)/$(BINARY).exe
 else
 	make genproto
-	go build ${LDFLAGS} -o $(CURR_DIR)/$(BINARY)
+	go build ${LDFLAGS} -o $(BIN_DIR)/$(BINARY)
 endif
 .PHONY: build
 
 hare:
-	cd cmd/hare ; go build -o $(CURR_DIR)/go-hare; cd ..
+ifeq ($(OS),Windows_NT)
```

Comment:
I think we should create a func build_path(NAME) and not repeat this if else when not necessary.

Reply:
or even use the if once to assign the correct path to $(BIN_DIR)

Reply:
Yeah, but you still need to push the name in the string (see the .exe for example).